tutor/src/components/notes.js

```diff
@@ -128,6 +130,7 @@ class NotesWidget extends React.Component {
     }
 
     if (this.highlighter) {
+      this.highlighter.eraseAll();
```

Comment:
This... sounds scary?

Reply:
eh, not really.  `eraseAll` just means to remove the highlights from the page, it doesn't actually send anything to server 😁